pss/pss.go

```diff
@@ -89,6 +95,7 @@ type Params struct {
 	privateKey          *ecdsa.PrivateKey
 	SymKeyCacheCapacity int
 	AllowRaw            bool // If true, enables sending and receiving messages without builtin pss encryption
```

Comment:
What about AllowRaw shouldn't beit a capability also?
BTW, raw is not the best name to describe non encryption. Maybe AllowCleartext.

Reply:
The `Raw` allow is per handler, not per node. And in any case, the days of the "Raw" concept are numbered, as the crypto revamp will make the node ignorant of the payload crypto scheme.